src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteWaypointModel.js

```diff
@@ -45,6 +47,38 @@ export default class StandardRouteWaypointModel extends BaseModel {
         }
 
         /**
+         * Maximum altitude at which to cross this waypoint
+         *
+         * @for StandardRouteWaypointModel
+         * @property altitudeMaximum
+         * @type {number}
+         */
+        this.altitudeMaximum = -1;
+
+        /**
+         * Minimum altitude at which to cross this waypoint
+         *
+         * @for StandardRouteWaypointModel
+         * @property altitudeMinimum
+         * @type {number}
```

Comment:
`@default`

instead of me adding this to all the missing ones, just go through and verify that these properties all have `@default` values commented